languages/erlang/exercises/concept/basics/.docs/after.md

```diff
@@ -0,0 +1,8 @@
+See the following sections of the Erlang documentation for more information:
```

Comment:
The after.md document should contain all information that is introduced in the introduction.md document (see [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsaftermd)).

Reply:
Should I paste in the information from `introduction.md` into `after.md`, or should I attempt to summarize it here?

Reply:
We've since updated the spec and the `after.md` is now replaced by a concept `about.md` document. For more information, see https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#concept-files

In short: the `after.md` document is no longer part of the exercise. Instead, each exercise will have a `concepts/<slug>/about.md` and `concepts/<slug>/links.json` file.

> Should I paste in the information from introduction.md into after.md, or should I attempt to summarize it here?

The docs can probably best explain this: https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#aboutmd